README.md

```diff
@@ -88,4 +88,15 @@ If you use the paraphrasing model (BART or GPT-2 fine-tuned on a paraphrasing da
   booktitle={Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing},
   year={2020}
 }
+```
+
+If you use MarianMT/ mBART/ T5 for translation task, or XLMR-LSTM model for Seq2Seq tasks, please cite [Localizing Open-Ontology QA Semantic Parsers in a Day Using Machine Translation](https://arxiv.org/abs/2010.05106) and the original paper that introduced the model.
+
+```bibtex
+@article{moradshahi2020localizing,
+  title={Localizing open-ontology QA semantic parsers in a day using machine translation},
+  author={Moradshahi, Mehrad and Campagna, Giovanni and Semnani, Sina J and Xu, Silei and Lam, Monica S},
+  journal={arXiv preprint arXiv:2010.05106},
+  year={2020}
+}
```

Comment:
Want to use the cite from the ACL anthology instead? https://www.aclweb.org/anthology/2020.emnlp-main.481/

Reply:
Sure